HIPs/incubation/hedera-hashgraph.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: default
+title: Hiero
```

Comment:
Should we change the name of the file too? And also replace `PROJECT` with Hiero?

Reply:
I will check if we can do that now.